### PR TITLE
[Fix] Embedded Videos don't work when users switch to the CK editor

### PR DESCRIPTION
**Pull request by edi-sipka**
## Describe the changes in this pull request
Ticket: https://app.shortcut.com/helpjuice/story/3147/embedded-videos-don-t-work-when-users-switch-to-the-ck-editor

- CKEditor was rendering only first match of id.
- Fix regex url.
- It is checking is there any additional query on url. 

### Screenshot (if this pull request changes the UI)

Screenshot 2023-05-24 at 18 15 28

### CODE REVIEW

**Review by diego-aslz — APPROVED** [review]

**Comment by NicholasJarr** [comment]
@edi-sipka 

Code is looks good to me, but there are two things you need to do before we can merge this PR:

1. Push the changes to the `hjeditor.js` and `hjeditor.map` to this PR. We usually push this to prevent conflicts on editor builds. Similar to #31.

2. Can you open a PR on our app repo with the new `hjeditor.js` and deploy it to staging so we can check it there?

**Comment by NicholasJarr** [comment]
@edi-sipka

I deployed changes to the editor on staging and I didn't notice you had deployed your changes there. So I ended up overriding your fix on staging. My bad. Can you open the PR on the app repo so I can use it to test your changes locally?

File: packages/ckeditor5-media-embed/src/mediaembedediting.ts
Comment (NicholasJarr) [inline packages/ckeditor5-media-embed/src/mediaembedediting.ts:1]: Can you fix the styles to be consistent with the rest of the file? If you're using VSCode, the ESLint plugin should show the warnings.

**Review by NicholasJarr — COMMENTED** [review]
I left a comment on the app PR regarding an issue I noticed on staging.

**Review by NicholasJarr — COMMENTED** [review]
Sorry for the late review. Code looks good now and it's working as expected. Can you rebase it with the latest changes so we can confirm all is working correctly?

**Comment by edi-sipka** [comment]
@NicholasJarr new PR is opened because it is supposed to be merged on helpjuice-build-v38.0.0 not on helpjuice-build-v2.

Here is PR: https://github.com/Helpjuice/ckeditor5/pull/54

**Pull request closed without merging**